internal/apis/certmanager/types_certificaterequest.go

```diff
@@ -26,13 +26,20 @@ const (
 	// Pending indicates that a CertificateRequest is still in progress.
```

Comment:
This file is a copy of what we have for the public API, I just removed the // +optional annotations and the json tags.